modules/programs/chromium.nix

```diff
@@ -134,9 +154,40 @@ let
         };
 
     in mkIf cfg.enable {
-      home.packages = [ cfg.package ];
-      home.file = listToAttrs (map extensionJson (cfg.extensions or [ ]));
+      home.packages = mkIf (cfg.package != null) [ cfg.package ];
+      home.file = mkIf (cfg.package != null)
+        (listToAttrs (map extensionJson cfg.extensions));
+    };
+
+  browserPkgs = {
```

Comment:
right, it is better to modularize such things. I didn't consider this before but I think simply calling `concatStringsSep " " list` is sufficient as it will return an empty string anyways if the list is empty, and optionalString effectively duplicates the work of checking the list. What do you think?